pyproject.toml

```diff
@@ -7,6 +7,8 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = ">=3.7,<4.0"
+typer = "^0.4.0"
+python-decouple = "^3.6"
```

Comment:
Since this dependency is no longer used, it should be removed.

Reply:
This was not resolved. This dependency should be removed since it's not used anywhere.